codec-http/src/main/java/io/netty5/handler/codec/http/websocketx/WebSocketClientProtocolConfig.java

```diff
@@ -343,8 +343,8 @@ public Builder dropPongFrames(boolean dropPongFrames) {
         }
 
         /**
-         * Handshake timeout in mills, when handshake timeout, will trigger user
-         * event {@link ClientHandshakeStateEvent#HANDSHAKE_TIMEOUT}
+         * Handshake timeout in millis, when handshake timeout, will trigger channel
+         * event {@link SslHandshakeCompletionEvent} with a {@link WebSocketHandshakeException}.
```

Comment:
If we are fixing `user event` in this PR, how about making it clear by saying `inbound channel event`?